### PR TITLE
Bump golang.org/x/text to v0.3.8

### PR DESCRIPTION
**Pull request by ameukam**
Signed-off-by: Arnaud Meukam <ameukam@gmail.com>

#### What type of PR is this?

/kind bug

#### Which issue(s) this PR fixes:
Fix for [CVE-2022-32149](https://osv.dev/vulnerability/GO-2022-1059)

#### Does this PR introduce a user-facing change?

```release-note
Updates golang.org/x/text to v0.3.8 to fix CVE-2022-32149
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: vendor/golang.org/x/text/language/parse.go [hunk 1]
```diff
@@ -147,6 +147,7 @@ func update(b *language.Builder, part ...interface{}) (err error) {
 }
 
 var errInvalidWeight = errors.New("ParseAcceptLanguage: invalid weight")
+var errTagListTooLarge = errors.New("tag list exceeds max length")
```
Comment (BenTheElder) [inline vendor/golang.org/x/text/language/parse.go:150]: notable

**Comment by BenTheElder** [comment]
https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/112989/check-dependency-stats/1579949460153176064
> Files /logs/artifacts/stats-base.json and /logs/artifacts/stats.json are identical

**Review by BenTheElder — COMMENTED** [review]
/lgtm
/approve

**Comment by dims** [comment]
A whole years worth of updates! https://github.com/golang/text/compare/v0.3.7...v0.3.8

/approve
/lgtm

**Pull request merged**